.github/workflows/repo_tests.yaml

```diff
@@ -105,6 +105,9 @@ jobs:
           - repo: trunk-io/plugins
             ref: main
 
+          - repo: trunk-io/trunk-action
+            ref: main
```

Comment:
can we somehow verify that the first pass of setup-node here will fail? or choose another repo where that's guaranteed?

i can totally see us pinning a valid engine in trunk-action, which is why this test makes me nervous